lib/user_agent/operating_systems/darwin.rb

```diff
@@ -55,7 +55,11 @@ module Darwin
         '21.2.0' => '15.2',   # also 15.2.1
         '21.3.0' => '15.3',   # also 15.3.1
         '21.4.0' => '15.4',   # also 15.4.1
-        '21.5.0' => '15.5',
+        '21.5.0' => '15.5',   # also 15.5.1
+        '21.6.0' => '15.6',
+        '22.0.0' => '16.0',   # also 16.0.1, 16.0.2 and 16.0.3
```

Comment:
```suggestion
        '22.0.0' => '16.0.x',
```
Probably it's better to use `16.0.x`? You will also need to change your spec accordingly.

Reply:
Thoughts on having a comment trailing the line? : `'22.0.0' => '16.0.x', # 16.0, 16.0.1, 16.0.2 and 16.0.3`

Reply:
I would prefer without the comment just to be consistent with the others, but I think either is fine. Thanks!

Reply:
Will remove - thanks!